pkg/operator/ceph/pool/controller.go

```diff
@@ -186,7 +187,7 @@ func createPool(context *clusterd.Context, p *cephv1beta1.Pool) error {
 
 	// create the pool
 	logger.Infof("creating pool %s in namespace %s", p.Name, p.Namespace)
-	if err := ceph.CreatePoolWithProfile(context, p.Namespace, *p.Spec.ToModel(p.Name), p.Name); err != nil {
+	if err := ceph.CreatePoolWithProfile(context, p.Namespace, *p.Spec.ToModel(p.Name), poolApplicationNameRBD); err != nil {
```

Comment:
this `createPool` function can be used for any generic pool that may never be used for RBD images.  What are the consequences of a pool having its app tag set for RBD, but never actually being used for RBD?

Reply:
Is that a common scenario, though (from what I understood, filesystems and object stores manage their pools on their own)?

Another solution would be to have an `applicationName` in a `Pool`'s spec, which would default to "rbd" if not set, but to me it seems a bit too involved for no particular reason.

Reply:
No, I do not believe it's a common scenario, but I'm curious as to what the consequences would be. Does Ceph treat the pool in a special way if it's been labeled for RBD? If so, does that special way affect usage of the pool that has nothing to do with RBD?

Reply:
To be completely honest, I didn't really understood what's the whole deal with application names. As I said in the [related issue](https://github.com/rook/rook/issues/2011), this affected me because in the dashboard you can only see block images that belong to pools with an application name set to "rbd".

Reply:
Thanks @lbogdan, that's fair that RGW/CephFS creation will also create their pools and tag them with the appropriate app tag.  I think defaulting to RBD is probably OK since that is very likely the expected usage of a pool not created via RGW/CephFS, and doesn't seem wildly out of line with the guidance at the end of https://ceph.com/community/new-luminous-pool-tags/.